python/custom_transformer/model.py

```diff
@@ -133,4 +133,5 @@ def postprocess(
         predictor_protocol=args.predictor_protocol,
         predictor_use_ssl=args.predictor_use_ssl,
     )
+    model.load()
```

Comment:
Might be a breaking change for those, who forgot to set `model.ready = True`

Reply:
This is the example transformer code, do we need to load the model ?

Reply:
This is for marking `model.ready=True`

Reply:
if the model is not marked true the server throws exception right?

Reply:
for rest yes, but for grpc it is not the case. I also noticed for rest transformer,  model is marked ready in `init method`, while grpc transformer is missing this. Probably, I will just mark the model ready in init method just like in the rest transformer.

Reply:
If user the user hits /model/ready endpoint, we can query the predictor.  This way we don't need this flag for transformer. https://github.com/kserve/kserve/pull/3348